configure

```diff
@@ -52,7 +52,7 @@ pie=1
 arch=`uname -s`
 cpu=`uname -m`
 
-GLIB_MIN_VERSION=2.14
+GLIB_MIN_VERSION=2.16
```

Comment:
Relevant thread in my personal issue tracker: https://github.com/dequis/bitlbee/issues/25 - first comment has a summary of the versions. Copypasting it:

2.12 - centos 5 *(currently works by disabling twitter module)*
**2.14 - current bitlbee requirement**
2.16 - debian 5.0 (lenny) / ubuntu 8.04 lts
**2.20 - current libpurple requirement**
2.24 - debian 6.0 (oldstable, squeeze) / ubuntu 10.04 lts
2.26 - centos 6
2.32 - debian 7.0 (stable, wheezy) / ubuntu 12.04 lts
2.36 - centos 7
2.40 - ubuntu 14.04 lts
2.42 - debian 8.0 (testing, jessie)

With this commit we're keeping support for "debian 5.0 (lenny) / ubuntu 8.04 lts" as some sort of coincidence, mostly because glib 2.16 is a very interesting release. But I'm totally in favor of bumping it up to 2.24. Centos 6 is also important, plenty of users with that thing.